configs/app/meta.ts

```diff
@@ -4,7 +4,7 @@ import { getEnvValue, getExternalAssetFilePath } from './utils';
 const defaultImageUrl = '/static/og_placeholder.png';
 
 const meta = Object.freeze({
-  promoteBlockscoutInTitle: getEnvValue('NEXT_PUBLIC_PROMOTE_BLOCKSCOUT_IN_TITLE') || 'true',
+  promoteBlockscoutInTitle: getEnvValue('NEXT_PUBLIC_PROMOTE_BLOCKSCOUT_IN_TITLE') === 'true',
```

Comment:
I assume this change will break the default behavior of the variable. If the variable is not passed, the value should be `true`. However, I agree that the current code has a bug. The proper fix would be:
```suggestion
  promoteBlockscoutInTitle: getEnvValue('NEXT_PUBLIC_PROMOTE_BLOCKSCOUT_IN_TITLE') === 'false' ? false : true,
```